lib/screens/repositories_screen.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+import '../widgets/drop_down.dart';
+
+class RepositoriesScreen extends StatefulWidget {
+  const RepositoriesScreen({super.key});
+
+  @override
+  State<RepositoriesScreen> createState() => _RepositoriesScreenState();
+}
+
+class _RepositoriesScreenState extends State<RepositoriesScreen> {
+  final List<String> _languages = [
```

Comment:
```suggestion
  final _languages = <String>[
```

Shorter, not required, just FYI

Reply:
done